clusterctl/clusterdeployer/clusterclient.go

```diff
@@ -164,15 +167,31 @@ func (c *clusterClient) WaitForClusterV1alpha1Ready() error {
 
 func (c *clusterClient) kubectlApply(manifest string) error {
 	r := strings.NewReader(manifest)
```

Comment:
I'm not sure why `r` is a separate variable instead of being inlined below (`cmd.Stdin = strings.NewReader(manifest)`).